### PR TITLE
P0 avatar  images

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added two components Responsive image and Avatar

### CODE REVIEW

File: docs/components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#avatar-container {
```
Comment (humzahasan) [inline docs/components/avatar/avatar.css]: container used for example in documentation so need not be added in component css but in a seperate CSS file for docuemtation site.

File: docs/components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 2rem;
+    gap: 2rem;
+}
+  
+.img-container {
```
Comment (humzahasan) [inline docs/components/avatar/avatar.css]: Same as above.

**Review by humzahasan — APPROVED** [review]
Incorporate the changes in future PR's post refactoring. Good to merge.

File: docs/components/css/typography.css [hunk 3]
```diff
@@ -14,6 +14,12 @@ h1,p {
     color: var(--color-dark-blue);
 }
 
+h3 {
+    text-align: center;
+    color: var(--color-dark-blue);
+    margin: 0.5rem;
```
Comment (iRohitGaur) [inline docs/components/css/typography.css]: suggestion: give this value in px if it is not going to change

File: docs/components/avatar/avatar.css (hunk 1 above)
Comment (iRohitGaur) [reply docs/components/avatar/avatar.css]: +1

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes 👍🏻

**Review by zoef-yt — APPROVED** [review]
Good to merge with above suggestions.

**Pull request merged**